src/datasources/cache/redis.cache.service.ts

```diff
@@ -42,9 +51,20 @@ export class RedisCacheService
     return await this.client.hGet(cacheDir.key, cacheDir.field);
   }
 
-  async deleteByKey(key: string): Promise<number> {
+  async deleteByKey(
+    key: string,
+    setInvalidationTime?: boolean,
```

Comment:
In which situations do we call this with `false`?

Reply:
I was thinking of giving the client of this class the ability to indicate if the `invalidationTimeMs` should be written. But maybe this can be simplified as there's no situation where this will be called with `false` value. 

I'm going to remove the optional parameter if you agree, so the calls are also simplified.

Reply:
I think we can assume that we will generate this entry every time. We can iterate on it if things change 👍

Reply:
Perfect, thanks, changed in [b92584c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/b92584cf5c50470b34305c8c9afffcc884c9fb21)